syft/sbom/format.go

```diff
@@ -18,20 +19,41 @@ func (f FormatID) String() string {
 	return string(f)
 }
 
+const AnyVersion = ""
```

Comment:
I feel that this belongs in the `format` package, so it would read `format.AnyVersion` instead of `sbom.AnyVersion`

Reply:
I forgot there is `formats` which is about capabilities and not definitions. I take back this comment